.gitignore

```diff
@@ -1,2 +1 @@
-target
-
+target
```

Comment:
add and empty new line

Reply:
The file still seems to miss the newline.

Reply:
Sorry, added now the endling newline